ngraph/core/src/function.cpp

```diff
@@ -266,7 +284,18 @@ std::vector<shared_ptr<ov::Node>> ov::Function::get_ordered_ops() const {
         nodes.push_back(param);
     }
 
-    return m_topological_sorter(nodes);
+    auto order = m_topological_sorter(nodes);
+
+    // Update nodes cache and update all nodes to have shared rt info
+    // which belongs to the current Function.
+    m_cached_ordered_ops.clear();
+    for_each(order.cbegin(), order.cend(), [this](const shared_ptr<Node>& node) {
+        m_cached_ordered_ops.push_back(node);
+        node->m_shared_rt_info.insert(m_shared_rt_info);
+    });
+    m_shared_rt_info->set_use_topological_cache(true);
```

Comment:
Can we also sort parameters/results/... to remove this core:
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/pass/serialize.cpp#L844
?